sklearn/gaussian_process/tests/test_gpr.py

```diff
@@ -652,3 +652,27 @@ def test_gpr_predict_error():
     err_msg = "At most one of return_std or return_cov can be requested."
     with pytest.raises(RuntimeError, match=err_msg):
         gpr.predict(X, return_cov=True, return_std=True)
+
+
+def test_y_std_with_multitarget_normalized():
+    """
+    Regression test for issues #17394 and #18065.
+    Check if GPR can compute y_std in predict() method when normalize_y==True
+    in multi-target regression.
+    """
+    X_train = np.random.rand((11, 10))
```

Comment:
Your test is failing as well: cf. [there](https://dev.azure.com/scikit-learn/scikit-learn/_build/results?buildId=32345&view=logs&jobId=32e2e1bb-a28f-5b18-6cfc-3f01273f5609&j=13650d7c-49e8-54f3-0598-c3480d1c1e4f&t=97f2162d-a0c4-54dc-9fce-2fceef7172e3)

Here it would be best to define a random state and then you can just use `randn`:

```python
rng = np.random.RandomState(42)

n_samples, n_features, n_targets = 12, 10, 6

X_train = rng.randn(n_samples, n_features)
y_train = rng.randn(n_samples, n_targets)
X_test = rng.randn(n_samples, n_features)
```